bin/headers.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+failure=0
+for file in $(find . -name "*.go"); do
+  head -n 1 $file | grep '^// Copyright 2017 Istio Authors' > /dev/null
```

Comment:
remove "Copyright 2017"? or use a regex to match "Copyright any_year Istio Authors"